cli/src/semgrep/meta.py

```diff
@@ -187,6 +187,7 @@ def to_dict(self) -> Dict[str, Any]:
             "commit_author_username": None,
             "commit_author_image_url": None,
             "commit_title": commit_title,
+            "commit_timestamp": self.commit_datetime,
```

Comment:
So this was already collected and defined and sent to the app as part of each Finding - do you agree? Looking at `RuleMatch.to_app_finding_format()`

Reply:
Yes, this was sent with each Finding, we just want to include it in the scan meta as well moving forward